docs/modules/releasenotes/partials/release-notes-25.1.adoc

```diff
@@ -197,3 +197,10 @@ image::lookup_column_world.png[]
 Configuration for selenium test setup is now done through scout properties instead of system properties.
 Refer to the migration guide for more details.
 
+== Migration Handler for data object deletion
+
+A new migration handler was implemented, which can be used when a data objects class is completely deleted +
```

Comment:
A new migration handler was implemented, which can be used -> The new `AbstractDoDeletionMigrationHandler` can be used